site/en/blog/longer-esw-lifetimes/index.md

```diff
@@ -35,9 +36,12 @@ Unfortunately, this behavior did not apply to extension events. Extension events
 
```

Comment:
Github won't let me comment on line 33, but:

> New service worker events received before the idle timer or long running task timer expire would reset the timers and extend the service worker's lifetime.

This should read "... would reset the idle timer and..." vs "reset the timers and".  The long-running task timer is not reset.

Reply:
Done.